### PR TITLE
Introduce classification example 

### PR DESCRIPTION
**Pull request by bbarnes52**
This example shows how to use the Mlflow classification recipe to train a classification model that can identify fraudulent credit card transactions. Information on the dataset available [here](https://www.kaggle.com/datasets/mlg-ulb/creditcardfraud), it is one of the datasets mounted to DBFS by default.

A few noteworthy items, some of which need to be addressed prior to landing this change:
- Currently the model trained is badly overfit to the training data.
- The dataset checked in in the `data/` folder is quite large by Git standards - just over 5MB. A smaller dataset will not suffice as it risks omitting positive examples from the eval/test sets during the split step, as the data is highly imbalanced.
- Running the example requires installing mlflow from HEAD (this is currently true of the regression example as well, if I'm not mistaken).

### CODE REVIEW

File: classification/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# providing a profile name such as `local` in the Recipe object constructor:
+# `p = Recipe(profile="local")`
+#
+# NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
+# files are properly formatted.
+
+recipe: "classification/v1"
+# Specifies the name of the column containing targets / labels for model training and evaluation
+target_col: "label"
+positive_class: 1
```
Comment (sunishsheth2009) [inline classification/recipe.yaml:16]: Can we explain what positive_class means here?

Comment (bbarnes52) [reply classification/recipe.yaml:17]: Done!

File: classification/profiles/databricks.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  #                 model to the MLflow Model Registry.
+  model_name: "credit_card_fraud_classifier"
+
+# Override the default train / validation / test dataset split ratios
+SPLIT_RATIOS: [0.75, 0.125, 0.125]
+
+INGEST_CONFIG:
+  # For different options please read: https://github.com/mlflow/mlp-classification-template#ingest-step
+  using: parquet
+  location: "dbfs:/databricks-datasets/credit-card-fraud/data/part-00000-tid-898991165078798880-9c1caa7b-283d-47c4-9be1-aa61587b3675-0-c000.snappy.parquet"
```
Comment (bbarnes52) [inline classification/profiles/databricks.yaml:21]: there are several data files, this is just one of them - can we use a regex to ingest the entire dataset

File: classification/steps/custom_metrics.py [hunk 3]
```diff
@@ -0,0 +1 @@
+# Created by MLflow Pipeliens
```
Comment (liangz1) [inline classification/steps/custom_metrics.py:1]: "Created by MLflow Recipes"

**Pull request merged**